src/build-extensions-container.sh

```diff
@@ -27,9 +24,5 @@ img=localhost/extensions-container
     --build-arg COSA=true --label version="$buildid" \
     -t "${img}" -f extensions/Dockerfile "${ctx_dir}")
 
-if [ -n "${cleanup_ctx_dir:-}" ]; then
-    rm -rf "${ctx_dir}"
-fi
-
```

Comment:
might be nice to still clean up the context dir though? Our [alias](https://github.com/coreos/coreos-assembler/blob/main/docs/building-fcos.md#define-a-bash-alias-to-run-cosa) still has `-v /var/tmp:/var/tmp` so this would cause developer dirs to get larger over time, right?

Reply:
The context dir is on the supermin disk which gets discarded anyway when we exit. So it didn't seem worth the effort to clean it up. Should I add a comment?

Reply:
oh, I see. No don't worry about it.